### PR TITLE
A few warning fixes

### PR DESCRIPTION
**Pull request by mattst88**

### CODE REVIEW

File: CUSTOMIZE.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Here's a minimal example:
 
 ```python
 # keybinding/keybinding.py
-class Keybinding(object):
+class Keybinding():
```
Comment (thehunmonkgroup) [inline CUSTOMIZE.md]: https://docs.python.org/3/tutorial/classes.html#class-definition-syntax seems to suggest we go even further and do `class Keybinding:`

Perhaps we should adjust the rest of these to match that syntax?

**Review by thehunmonkgroup — CHANGES_REQUESTED** [review]
Overall these seem like easy and sensible changes, and should be easy to get in once you address the one question listed in the review.

File: CUSTOMIZE.md (hunk 1 above)
Comment (mattst88) [reply CUSTOMIZE.md]: Ah, good point. Yes, I'll make that change.

**Comment by thehunmonkgroup** [comment]
Thanks for your work on this!

**Comment by mattst88** [comment]
Thank you for the awesome software!

**Pull request merged**